src/index.js

```diff
@@ -202,6 +208,24 @@ class HCaptcha extends React.Component {
       if (onError) onError(event);
     }
 
+    handleOpen () {
+      if (this.props.onOpen) {
+        this.props.onOpen();
+      }
+    }
+
+    handleClose () {
+      if (this.props.onClose) {
+        this.props.onClose();
+      }
+    }
+
+    handleChallengeExpired () {
+      if (this.props.onChalExpired) {
+        this.props.onChalExpired();
+      }
+    }
+
```

Comment:
Please also consider using `if (!isApiReady || isRemoved) return` in these callbacks.